java/send-http-request/README.md

```diff
@@ -0,0 +1,69 @@
+# 📧 Send HTTP request
+
+A Java Cloud Function for sending http requests.
+
+_Example input:_
+
+```json
+{
+  "url": "https://google-translate1.p.rapidapi.com/language/translate/v2",
+  "method": "POST",
+  "headers": {
+    "X-RapidAPI-Key": "KEY",
+    "X-RapidAPI-Host": "HOST"
+  },
+  "body": {
+    "q": "Ahoj světe!",
+    "target": "en",
+    "source": "cs"
+  }
+}
+```
+
+_Example output:_
+
+```json
+{
+    "success":true,
+    "response":"{\"data\":{\"translations\":[{\"translatedText\":\"Hello world!\"}]}}"
+}
+```
+
+## 📝 Input Description
+
+List of input used by this cloud function:
+
+* **url** - URL where request should be sent to 
+* **method** - Which method should be sent
+* **headers** [optional] - should be string string key value pairs
+* **body** [optional] - should be string string key value pairs
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd java/send-http-request
+```
+
+2. Enter this function folder and build the code:
+   
+   ```
+   docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts --rm --interactive --tty --volume $PWD:/usr/code openruntimes/deno:1.14 sh /usr/local/src/build.sh
```

Comment:
Teehee, it looks like this might be the deno runtime again. Can you update the instructions here so I can try your java example?

Reply:
Updated. Thanks